VSRAD.Package/Server/CommunicationChannel.cs

```diff
@@ -85,9 +85,21 @@ public CommunicationChannel(SVsServiceProvider provider, IProject project)
                 options.PropertyChanged += (s, e) => { if (e.PropertyName == nameof(options.ActiveProfile)) ForceDisconnect(); };
         }
 
-        public async Task SendAsync(ICommand command)
+        public async Task SendAsync(ICommand command) => await SendAsync(command, true);
```

Comment:
`async` is not required here, you can just return the task:
`public Task SendAsync(ICommand command) => SendAsync(command, true);`

Reply:
Thanks, fixed in 49b5022